.github/workflows/visual-regression-tests.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    types:
+      - ready_for_review
+
+jobs:
+  visual-regression-test-components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12"
```

Comment:
Note to self - see if this can be updated to 14 LTS

Reply:
It might be best to keep it on 12 anyway since that is the Node version on GOV.UK CI.

Reply:
Good to know - I'll keep it on 12. Ta!